### PR TITLE
Add open cash drawer command

### PR DESCRIPTION
**Pull request by mdsiddiq-till**
## Description

Add open cash drawer command

## Related Issues

Issue: #XXX

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/buffer-builder.ts [hunk 1]
```diff
@@ -182,6 +182,18 @@ export class BufferBuilder {
     return this;
   }
 
+  /**
+   * Register open cash drawer action
+   * @return BufferBuilder
+   */
+  public openCashDrawer(): BufferBuilder {
+    // kick drawer 1, pin 2
```
Comment (lakshmaji-till) [inline src/buffer-builder.ts:190]: I think we can listen to drawer status too!

File: src/buffer-builder.ts [hunk 2]
```diff
@@ -182,6 +182,18 @@ export class BufferBuilder {
...
+    this.buffer.write(Command.CD_KICK_1);
+    // kick drawer 2, pin 5
+    this.buffer.write(Command.CD_KICK_2);
```
Comment (lakshmaji-till) [inline src/buffer-builder.ts:193]: `CD_KICK_1` 😬 (can we rename it)

File: src/buffer-builder.ts (hunk 1 above)
Comment (lakshmaji-till) [reply src/buffer-builder.ts:190]: i dont recall exact status it will be something like `ALB`!

File: src/buffer-builder.ts (hunk 2 above)
Comment (mdsiddiq-till) [reply src/buffer-builder.ts:193]: It is like 1st cash drawer kick 
and 2nd cash drawer kick

File: src/buffer-builder.ts (hunk 1 above)
Comment (mdsiddiq-till) [reply src/buffer-builder.ts:190]: I don't get it.
This command sends a pulse to cash drawer.

Comment (mdsiddiq-till) [reply src/buffer-builder.ts:190]: You want it not to send pulse if it is already opened?

**Pull request merged**